scripts/generate_openapi_document.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+from unittest.mock import (
+    PropertyMock,
+    patch,
+)
+
+from azul import (
+    config,
+)
+from azul.files import (
+    write_file_atomically,
+)
+from azul.modules import (
+    load_app_module,
+)
+
+
+def main():
+    with patch.object(target=type(config),
+                      attribute='catalogs',
+                      new_callable=PropertyMock,
+                      return_value={
+                          'dcp2': config.Catalog(name='dcp2',
+                                                 atlas='hca',
+                                                 plugins=dict(repository='tdr',
+                                                              metadata='hca'))
+                      }):
```

Comment:
This would probably look better if you extracted the dict literal into a variable.

Reply:
Adding the assert from your experiments wouldn't hurt either.

Reply:
@danielsotirhos 👀